File: java/1.tutorial/array.java
public class arrays {
  public static void main(String[] args) {
    // this is how we make an array
    String[] friends = {"Bethany", "Jake", "Ben", "Parker"};
    System.out.println(friends[3]);
    // this is how to print the length of an array
    System.out.println(friends.length);
    // this is how to print an entire array
    for (int i = 0; i < friends.length; i++) {
      System.out.println(friends[i]);
    }
  }
 }

File: cs/1.tutorial/array.cs
using System;

public class Array // arrayer? i hardly know her!
{
  public static void Main(string[] args)
  {
    string[] friends = {"Bethany", "Jake", "Ben", "Kaylen", "G-Dawg"};
    Console.WriteLine(friends[0]); // outputs "Bethany"
    Console.WriteLine(friends[4]); // outputs "G-Dawg"
    // changing an array item
    friends[4] = "Parker";
    Console.WriteLine(friends[4]);
    // lengths
    Console.WriteLine(friends.Length); // outputs 5
  }
}
